src/cfnlint/data/CloudSpecs/us-east-1.json

```diff
@@ -94558,7 +94088,7 @@
         },
         "Parameters": {
           "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-association.html#cfn-ssm-association-parameters",
-          "ItemType": "ParameterValues",
+          "ItemType": "List",
```

Comment:
@kddejong calling this out since [VS Code extension update failed due to this change](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/runs/2422790561?check_suite_focus=true#step:5:16961), and this property type has caused issues in the past. Probably worth looking into